scripts/create-instance.sh

```diff
@@ -107,15 +107,15 @@ set -o errexit
 set -o nounset
 
 if [[ "$@" != *"--devel"* ]]; then
-# sphinxdoc-install-invenio-full-begin
-# FIXME the next pip commands are needed only for invenio<3.0.0a3
-pip install invenio-db[postgresql] --pre
-pip install invenio-access[postgresql] --pre
-pip install invenio-search --pre
-pip install dojson --pre
-# now we can install full Invenio:
-pip install invenio[full] --pre
-# sphinxdoc-install-invenio-full-end
+    # sphinxdoc-install-invenio-full-begin
+    # FIXME the next pip commands are needed only for invenio<3.0.0a3
+    pip install invenio-db[postgresql] --pre
+    pip install invenio-access[postgresql] --pre
+    pip install invenio-search --pre
+    pip install dojson --pre
+    # now we can install full Invenio:
+    pip install invenio[full] --pre
+    # sphinxdoc-install-invenio-full-end
 else
```

Comment:
The indentation was intentionally left like this, so that the generated Sphinx documentation would  look good. Have you checked it out?